scene/2d/animated_sprite_2d.cpp

```diff
@@ -660,11 +660,15 @@ StringName AnimatedSprite2D::get_animation() const {
 }
 
 String AnimatedSprite2D::get_configuration_warning() const {
+	String warning = Node2D::get_configuration_warning();
+
 	if (frames.is_null()) {
-		return TTR("A SpriteFrames resource must be created or set in the \"Frames\" property in order for AnimatedSprite to display frames.");
+		if (!warning.empty())
```

Comment:
We're in the middle of adapting our style guide to enforce using braces around single-statement if blocks, could you amend this commit to do so? `clang-format` can't enforce it sadly.